kvirt/cli.py

```diff
@@ -3152,6 +3154,9 @@ def cli():
     ibmhostcreate_parser = hostcreate_subparsers.add_parser('ibm', help=ibmhostcreate_desc,
                                                             description=ibmhostcreate_desc)
     ibmhostcreate_parser.add_argument('--iam_api_key', help='IAM API Key', metavar='IAM_API_KEY', required=True)
+    ibmhostcreate_parser.add_argument('--access_key_id', help='Access Key Id', metavar='ACCESS_KEY_ID', required=True)
```

Comment:
i think those keys should be optional, since one can use ibm cloud without s3 interaction at all